src/dsp/agc_impl.h

```diff
@@ -34,7 +34,7 @@ class CAgc
 public:
     CAgc();
     virtual ~CAgc();
-    void SetParameters(bool AgcOn, bool UseHang, int Threshold, int ManualGain, int Slope, int Decay, double SampleRate);
+    void SetParameters(bool AgcOn, bool UseHang, int Threshold, int ManualGain, int Slope, int Decay, float SampleRate);
```

Comment:
`SampleRate` is assigned to the `m_SampleRate` member variable, which is a `float`. So there's not much point receiving it as a `double`.